src/main/java/com/commercetools/sync/inventory/helpers/InventorySyncStatistics.java

```diff
@@ -0,0 +1,55 @@
+package com.commercetools.sync.inventory.helpers;
+
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+
+import static java.lang.String.format;
+
+/**
+ * Statistics of inventory synchronisation process
+ */
+public class InventorySyncStatistics extends BaseSyncStatistics {
+
+    private int unprocessedDueToEmptySku;
+
+    public InventorySyncStatistics() {
+        super();
+    }
+
+    /**
+     * Get the total number of resources that weren't processed because they had empty sku.
+     *
+     * @return total number of resources unprocessed due to empty sku.
+     */
+    public int getUnprocessedDueToEmptySku() {
+        return unprocessedDueToEmptySku;
+    }
+
+    /**
+     * Increment the total number of resources that weren't processed because they had empty sku.
+     */
+    public void incrementUnprocessedDueToEmptySku() {
+        this.unprocessedDueToEmptySku++;
+    }
+
+    /**
+     * Builds a summary of the inventory sync statistics instance that looks like the following example:
+     * <pre>
+     *     Summary of inventory synchronisation:
+     *     2 inventory entries weren't processed due to empty sku.
+     *     25 inventory entries were processed in total (9 created, 5 updated, 2 failed to sync).
```

Comment:
@andrii-kovalenko-ct Here's the concept: "processed" means that draft was taken and checked if its sync is needed. When there exists in CTP analogue entry with same sku, supply channel etc. as in passed draft then no action is needed and only "processed" counter is incremented.
I will change above line to be explicit and less confusing:
` 25 inventory entries were processed in total (9 created, 5 updated, 2 failed to sync, 9 already up to date)`